src/main/java/org/generationcp/middleware/dao/LocationDAO.java

```diff
@@ -589,4 +608,33 @@ public Location getDefaultLocationByType(final Integer type) {
 		return Optional.empty();
 	}
 
+	public boolean blockIdIsUsedInFieldMap(final List<Integer> blockIds) {
+		try {
+
+			final StringBuilder queryString = new StringBuilder().append("SELECT ");
```

Comment:
@cuenyad 1) use a constant 2) dont we have constants fro 1155 and 8583?

Reply:
done.